ksql-parser/src/main/antlr4/io/confluent/ksql/parser/SqlBase.g4

```diff
@@ -86,12 +86,24 @@ resultMaterialization
     : CHANGES
     ;
 
+schemaTableElements
+    : '(' schemaTableElement (',' schemaTableElement)* ')'
+    ;
+
 tableElements
-    : '(' tableElement (',' tableElement)* ')'
+    : '(' statementTableElement (',' statementTableElement)* ')'
+    ;
+
+schemaTableElement
+    : (identifier '.')? tableElement (KEY)?
+    ;
+
+statementTableElement
+    : tableElement (KEY)?
     ;
 
 tableElement
-    : identifier type (KEY)?
+    : identifier type
```

Comment:
Do we need to drop `(KEY)?` from `tableElement`? Can we not just have:

```
schemaElements
    : '(' schemaElement (',' schemaElement)* ')'
    ;
    
schemaElement
    : (identifier '.')? tableElement
    ;

tableElements
   : '(' tableElement (',' tableElement)* ')'
   
tableElement
   : identifier type (KEY)?
```

And have both `visitStatementTableElement` and `visitTableElement` in the `AstBuilder`?